estuary/templates/deployment.yaml

```diff
@@ -36,6 +36,23 @@ spec:
             value: {{ .Values.estuary.data_topic }}
           - name: DLQ_TOPIC
             value: {{ .Values.estuary.dlq_topic }}
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: NODE_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: NODE_ID
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: LIVEVIEW_SALT
+            valueFrom:
+              secretKeyRef:
+                name: estuary_lv_salt
```

Comment:
this name is invalid; it needs to match the one you created in the estuary deploy repo: "estuary-lv-salt" without the underscores.